packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -336,7 +336,7 @@ const testProjectAnswers = () => {
 const demoProjectAnswers = () => {
     const config = {
         projectName: 'demo-storefront',
-        instanceUrl: 'https://zzte-053.sandbox.us02.dx.commercecloud.salesforce.com/',
```

Comment:
i don't know if removing the ending `/` breaks anything?

Reply:
I think we're good. I tested the project locally, tests are passing and we were not adding the trailing slash in our test preset.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7b972eeba840c62e622dadef65f3f8564d69aa1c/packages/pwa-kit-create-app/scripts/create-mobify-app.js#L324